### PR TITLE
Rln relay key generation

### PR DESCRIPTION
**Pull request by staheri14**
Addresses https://github.com/status-im/nim-waku/issues/382

### CODE REVIEW

File: tests/v2/test_waku_rln_relay.nim [hunk 1]
```diff
@@ -206,4 +206,15 @@ suite "Waku rln relay":
...
+    var key = membershipKeyGen()
+    var empty : array[32,byte]
+    check:
+      key.isSome
+      key.get().secretKey.len == 32
+      key.get().publicKey.len == 32
+      key.get().secretKey != empty
+      key.get().publicKey != empty
+    
+    debug "the generated membership key pair: ", key
```
Comment (oskarth) [inline tests/v2/test_waku_rln_relay.nim:220]: newline (editor setting?)

File: tests/v2/test_waku_rln_relay.nim [hunk 2]
```diff
@@ -206,4 +206,15 @@ suite "Waku rln relay":
       check:
         # the public and secret keys together are 64 bytes
         generatedKeys.len == 64
-      debug "generated keys: ", generatedKeys 
\ No newline at end of file
+      debug "generated keys: ", generatedKeys 
+  test "membership Key Gen":
```
Comment (oskarth) [inline tests/v2/test_waku_rln_relay.nim:210]:
```suggestion
      debug "generated keys: ", generatedKeys 
      
  test "membership Key Gen":
```

**Review by oskarth — APPROVED** [review]

File: tests/v2/test_waku_rln_relay.nim (hunk 1 above)
Comment (staheri14) [reply tests/v2/test_waku_rln_relay.nim:220]: are you suggesting me to add newline?

Comment (staheri14) [reply tests/v2/test_waku_rln_relay.nim:221]: I think you meant to delete the newline because there was already a newline

**Pull request merged**